classes/Commands/AbstractCommand.php

```diff
@@ -81,6 +81,8 @@ protected function setupEnvironment(InputInterface $input, OutputInterface $outp
         define('_PS_ADMIN_DIR_', $adminDir);
 
         $this->upgradeContainer = new UpgradeContainer($prodRootDir, $adminDir);
+        $this->upgradeContainer->getLogsState()->setTimeZone(date_default_timezone_get());
```

Comment:
```suggestion
        // We need to store the timezone this early because it can be altered by the core initialization later.
        $this->upgradeContainer->getLogsState()->setTimeZone(date_default_timezone_get());
```